webapp/src/pages/EditUserPage/EditUserPage.css

```diff
@@ -64,15 +64,3 @@
     grid-column: 1 / var(--column-end);
   }
 }
-
```

Comment:
Why are you changing this page now?

Reply:
Sorry, not my intention. I was trying things because I've used as a reference and I forgot to undo changes. I will revert it